src/js/menu/media-rendition-menu.ts

```diff
@@ -36,6 +41,37 @@ class MediaRenditionMenu extends MediaChromeMenu {
     ];
   }
 
+  static override formatMenuItemText(
+    text: string,
+    rendition?: Rendition
+  ): string {
+    return super.formatMenuItemText(text, rendition);
+  }
+
+  static formatRendition(
+    rendition: Rendition,
+    { showBitrate = false }: FormatRenditionOptions = {}
+  ): string {
+    const renditionText = `${Math.min(
+      rendition.width as number,
+      rendition.height as number
+    )}p`;
+
+    if (showBitrate && rendition.bitrate) {
+      const mbps = rendition.bitrate / 1000000;
+      const bitrateText = `${mbps.toFixed(mbps < 1 ? 1 : 0)} Mbps`;
+      return `${renditionText} (${bitrateText})`;
+    }
+
+    return this.formatMenuItemText(renditionText, rendition);
+  }
+
+  static compareRendition(a: Rendition, b: Rendition): number {
+    return b.height === a.height
+      ? (b.bitrate ?? 0) - (a.bitrate ?? 0)
+      : b.height - a.height;
+  }
```

Comment:
Formats in descending order by height then bitrate. It can be overridden for custom sorting.